### PR TITLE
Review/html css

### PR DESCRIPTION
**Pull request by zhaol**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        <td>April</td>
+        <td>$0</td>
+    </tr>
+</table>
+<head>
+    <h2>An Igloo</h2>
+    <p>An igloo is made of <span style="color:blue;font-family:Times New Roman">ice</span>.</p>
+    <img src="https://i.pinimg.com/originals/c2/c9/c0/c2c9c0b730a0933a45e0bd120a55fd57.jpg" style="width: 500px; height:600px" alt="igloo.jpg">
+    
+</head>
```
Comment (zhaol) [inline index.html]: The `<head>` tag is usually not nested inside the `<body>` tag
For example:
```
<html>
  <head></head>
  <body></body>
</html>
```
is more typical than:
```
<html>
  <body>
      <head></head>
  </body>
</html>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+
+
+<section>
```
Comment (zhaol) [inline index.html]: I like the use of the `<section>` tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<div class ="header">
+<style>
+table, th, td {
+    border: 1px solid black;
+}
+</style>
+</div class="header">
```
Comment (zhaol) [inline index.html]: Attributes are usually on the opening tag and not the closing tag
This:
`<div class="header"></div>`
is more conventional than this:
`<div></div class="header">`

File: index.html [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+<div id ="nav">
```
Comment (zhaol) [inline index.html]: There's a `<nav>` tag that can be more semantically appropriate: https://www.w3schools.com/tags/tag_nav.asp

File: index.html [hunk 5]
```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+
+<div class ="header">
```
Comment (zhaol) [inline index.html]: There's a `<header>` tag that can also be used inside the `<body>` tag.

But I think `<style>` tags are typically nested inside `<head>` tags like this:
```
<html>
  <head>
    <style></style>
  </head>
  <body></body>
</html>
```

File: index.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+<section>
+<p><b>Disadvantages of Igloo's</b></p> <!-- List of unordered items-->
+<ul>
+    <li>None</li>
+    <li>None</li>
+    <li>None</li>
+</ul>
+</section>
+
+<footer>
```
Comment (zhaol) [inline index.html]: I like the use of the `<footer>` tag

**Review by zhaol — COMMENTED** [review]
Good job on you first web page 👍 

Here are some overall comments:
The entire file is indented a bit weird for me.  This is more of what I'm used to:
```
<html>
  <head>
    <title>...</title>
    <style>...</style>
  </head>
  <body>
    <table>...</table>
    <p>...</p>
    <ol>...</ol>
    <ul>...</ul>
  </body>
</html>
```

Also, it will be good to try out the external stylesheet way of including CSS.

Other than that thanks for sharing and keep up the good work.  😀

File: index.html (hunk 3 above)
Comment (optimizasean) [reply index.html]: Usually for headers we will use `<hX>` where X is a number 1-6 based on header value with 1 as the primary for things likes titles and 6 the smallest.

File: index.html (hunk 4 above)
Comment (optimizasean) [reply index.html]: bootstrap also has a nav class if you are using that framework, in which case it is best to combine `<nav id="nav" class="nav ...">...</nav>`.  Id should not be necessary I think, also nav as an id is not very descriptive, maybe primary-navbar, title-navbar, main-navigation-sidebar or so on.

File: index.html (hunk 5 above)
Comment (optimizasean) [reply index.html]: For the header tag, I don't think I know the good practice of what a header would be.  Should headings `<h1>`, `<h2>`, ... be used instead or is there a clear difference between header(s) and headings?

Comment (optimizasean) [reply index.html]: What should be contained in the header?

**Comment by zhaol** [comment]
gonna close this merge/pull request since it is obsolete.  Hope all is well :)

**Pull request closed without merging**